cockroachcloud/cmek-ops-gcp.md

```diff
@@ -9,6 +9,18 @@ This page covers the procedures required to provision a Customer Managed Encrypt
 
 This is part of the larger process of [Enabling CMEK for a {{ site.data.products.dedicated }} cluster](managing-cmek.html#enable-cmek).
 
+## Overview
+
+- In [Step 1. Provision the cross-tenant service account](#step-1-provision-the-cross-tenant-service-account), we will create a service account that will be used by {{ site.data.products.dedicated }} to access the CMEK key.
```

Comment:
Perhaps this sentence could be adjusted to "... we will create a service account that {{ site.data.products.dedicated }} will use to access the CMEK key." To remove the passive.